### DISCUSSION TITLE
standardize global variable name about UI

### DISCUSSION THREAD

**Discussion by Freed-Wu** [neovim/neovim#31366] [Ideas]
Now neovim have many UI, some of them set some variable to allow user to do some thing in vimrc. such as firenvim's `g:started_by_firenvim`, vscode-neovim's `g:vscode`, ...

How about standardize the global variable name? such as `g:ui_id == 'firenvim'`, 'vscode', ...? Or any official standard about UI to make it more regular?

**Comment by clason** [comment]
We are not in the business of standardizing other projects. Best course of action is to contact the GUI devs and suggest that they come up with a standard that makes sense _for them_.
